homeassistant/components/light/blinksticklight.py

```diff
@@ -0,0 +1,64 @@
+"""
```

Comment:
This is probably my python ignorance coming through, but I was getting errors with both the blinkstick library and my own file being called "blinkstick".  Changing my file fixed this, but I appreciate it's likely not the Right Way.